detect.go

```diff
@@ -23,16 +24,23 @@ type BuildPlanMetadata struct {
 // and requires cpython and pip at build.
 func Detect() packit.DetectFunc {
 	return func(context packit.DetectContext) (packit.DetectResult, error) {
-		_, requirementEnvExists := os.LookupEnv("BP_PIP_REQUIREMENT")
-		defaultRequirement := "requirements.txt"
+		requirementsFile := "requirements.txt"
+		envRequirement, requirementEnvExists := os.LookupEnv("BP_PIP_REQUIREMENT")
+		if requirementEnvExists {
+			requirementsFile = envRequirement
+		}
 
-		defaultRequirementExists, err := fs.Exists(filepath.Join(context.WorkingDir, defaultRequirement))
-		if err != nil {
-			return packit.DetectResult{}, err
+		anyRequirementsFileExists := false
+		for _, filename := range strings.Split(requirementsFile, " ") {
+			found, err := fs.Exists(filepath.Join(context.WorkingDir, filename))
+			if err != nil {
+				return packit.DetectResult{}, err
+			}
+			anyRequirementsFileExists = anyRequirementsFileExists || found
```

Comment:
Shouldn't it use `&&`? Pip install will fail if any of the specified requirements is **missing**.

```
$ mkdir example && cd example && touch a.txt
$ PIP_REQUIREMENT='a.txt b.txt' pip install
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'b.txt'
```

Reply:
Oh, perfect. I must have misread your previous PR where you introduced support for multiple requirements files.

I'll fix that up.

Reply:
I inverted the logic in https://github.com/paketo-buildpacks/pip-install/pull/266/commits/27fe53345fd97069e1911198d41008a15b62a3ff so I'll resolve this comment.

Thanks for catching this and calling it out!